src/constants/errors/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export const INVALID_FORM_NAME:string = '0';
+export const ENFORCE_FAILED:string = '1';
+export const EXPECT_TYPE_FAILURE:string = '2';
+export const PASSABLE_ARGS_NO_ARGS:string = '3';
+export const PASSABLE_ARGS_UNEXPECTED_ARGS_1:string = '4';
+export const PASSABLE_ARGS_UNEXPECTED_ARGS_2:string = '5';
+export const PASSABLE_ARGS_UNEXPECTED_ARGS_3:string = '6';
+export const PASSABLE_ARGS_UNEXPECTED_ARGS_4:string = '7';
```

Comment:
It's purely semantical, as the error naming is just an inner convention, but categorizing the errors, and assigning each category a prefix would make adding errors and debugging nicer in the future.
For example, all argument-related errors are between 100-200.

Reply:
This is a good idea, though, not in the scope of this PR. This should be handled separately.